### ISSUE TITLE
Tests fail under GAP master with all packages loaded

### ISSUE THREAD

**Issue by alex-konovalov** [gap-packages/xmod#59]
See e.g. https://travis-ci.org/gap-infra/gap-docker-pkg-tests-master/jobs/497783640

```
testing: /home/gap/inst/gap-master/pkg/XMod-2.72/tst/gp3objmap.tst
########> Diff in /home/gap/inst/gap-master/pkg/XMod-2.72/tst/gp3objmap.tst:
145
# Input is:
xsC16 := CrossedSquareOfCat2Group( C16 );
# Expected output:
crossed square with crossed modules:
      up = [Group( [ (1,5)(2,6)(3,7)(4,8) ] ) -> Group( [ ( 2, 6)( 4, 8) ] )]
    left = [Group( [ (1,5)(2,6)(3,7)(4,8) ] ) -> Group( 
[ (1,2,3,4)(5,6,7,8), (), () ] )]
    down = [Group( [ (1,2,3,4)(5,6,7,8), (), () ] ) -> Group( () )]
   right = [Group( [ ( 2, 6)( 4, 8) ] ) -> Group( () )]
# But found:
crossed square with crossed modules:
      up = [Group( [ (), (1,5)(2,6)(3,7)(4,8) ] ) -> Group( 
[ (), (), (2,6)(4,8) ] )]
    left = [Group( [ (), (1,5)(2,6)(3,7)(4,8) ] ) -> Group( 
[ (1,2,3,4)(5,6,7,8), (), () ] )]
    down = [Group( [ (1,2,3,4)(5,6,7,8), (), () ] ) -> Group( [ () ] )]
   right = [Group( [ (), (), (2,6)(4,8) ] ) -> Group( [ () ] )]
########
```

Perhaps you can make this test more robust by suppressing the exact output and testing some of its properties instead.

**Comment by cdwensley** [comment]
It’s not clear to me which of the following you are asking for: 
(a)  just change gp3objmap.tst in the development version, or 
(b)  make a change and then a new release. 

I’m guessing the latter since the test is running XMod 2.72, and the offending command, specified as line 145, is 176 in the development version. 
The crossed square material was undergoing a major rewrite back in January, and this has been put to one side while I work on the IBNP package, 
so it will not be a simple matter to make a new release - but I could have a go, if necessary.

**Comment by alex-konovalov** [comment]
Well, if you will simply update the output to the new look and make a new release, then tests will pass with the GAP master, but will be broken in the GAP stable. So ideally you need to redesign the test to something like e.g.
```
gap> xsC16 := CrossedSquareOfCat2Group( C16 );;
gap> IdGroup(Source(xsC16.up));
<id_group>
```
which will pass in both settings.

**Comment by cdwensley** [comment]
Fixed in release 2.73.
